<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialsCatalog</class>
 <widget class="QWidget" name="MaterialsCatalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materials Setting</string>
  </property>
  <property name="toolTip">
   <string>材料设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Materials Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>材料</string>
        </property>
        <property name="text">
         <string>Material：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="comboBoxMaterial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select a Material from the Catalog</string>
        </property>
        <property name="whatsThis">
         <string>Select a Material from the Catalog to view/edit exposure timing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QTableWidget" name="tableWidgetParameters">
        <property name="toolTip">
         <string>材料参数</string>
        </property>
        <property name="whatsThis">
         <string>Exposure settings for a given material and xy pixel size.  Each possible layer thickness and it's associated timing is shown.</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string>20μm</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>50μm</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>100μm</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Attach Exposure(sec)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time of Exposure(sec)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Intensity of Exposure(0-255)</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>添加</string>
        </property>
        <property name="whatsThis">
         <string>Add a new Material to the Catalog</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>移除</string>
        </property>
        <property name="whatsThis">
         <string>Remove the selected material from the Catalog.  Default entries can not be deleted.</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonSave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>保存</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>取消</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>MaterialsCatalog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
